### ISSUE TITLE
Transmission plugin broken, can't download: Object of type bytes is not JSON serializable

### ISSUE THREAD

**Issue by antivirtel** [Flexget/Flexget#3724]
### Expected behaviour:

That it adds torrents to Transmission. 

### Actual behaviour:
It crashes, and doesn't add the torrents or notify me (via another plugin after it). 

### Steps to reproduce:
- Step 1: configure the transmission plugin to download

#### Config:
```yaml

schedules:
- tasks: '*'                                                         interval:
      minutes: 4

tasks:
  shows:
    rss:                                                             
      url: "https://foo.com/rss.php?key={? var.foo.passkey ?}"
      headers:                                                            
        Cookie: "nick={? var.foo.nick ?}; pass={? var.foo.pass ?}"
    series: 
      shows:
        - foo
      settings:
         shows:
           upgrade: yes
           timeframe: 5 days
           target: h264 1080p+ webdl+
           quality: h264|h265 360p+
      transmission:
        host: localhost
        port: 9091                                                              
        username: '{? var.transmission.user ?}'
        password: '{? var.transmission.passwd ?}'
        main_file_only: yes
        main_file_ratio: 0.85
        include_subs: yes
        path: "/path/{{ series_name }}"
        add_paused: yes
```
  
#### Log:
<details>
 <summary>(click to expand)</summary>

```
File "/usr/lib/python3.9/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type bytes is not JSON serializable
```
</details>

### Additional information:

- FlexGet version: 3.5.33
- Python version: 3.9.5
- Installation method: pip
- Using daemon (yes/no): yes, via SystemD
- OS and version: Ubuntu 22.04
- Link to crash log: https://pastebin.com/uFgUB4RP
- installed pip packages:
<details>
 <summary>(click to expand)</summary>

```
$ pip freeze
aiohttp==3.8.1
aiosignal==1.2.0
alabaster==0.7.12
async-timeout==4.0.2
attrs==21.2.0
Automat==20.2.0
Babel==2.9.1
blinker==1.4
certifi==2021.10.8
cffi==1.15.0
chardet==4.0.0
charset-normalizer==2.0.9
chkcrontab==1.7
click==8.0.3
cloud-init==23.1.1
colorama==0.4.4
command-not-found==0.3
configobj==5.0.6
constantly==15.1.0
cryptography==36.0.1
dbus-python==1.2.18
distro==1.6.0
distro-info==1.0
dnspython==1.16.0
docutils==0.17.1
entrypoints==0.3
fail2ban==0.11.1
frozenlist==1.2.0
gpg===1.13.1-unknown
httplib2==0.20.2
hyperlink==21.0.0
idna==3.3
imagesize==1.3.0
importlib-metadata==4.10.0
incremental==21.3.0
iotop==0.6
jeepney==0.7.1
Jinja2==3.0.3
jsonpatch==1.32
jsonpointer==2.2
jsonschema==4.3.2
keyring==23.4.0
language-selector==0.1
launchpadlib==1.10.15.1
lazr.restfulclient==0.14.4
lazr.uri==1.0.6
Mako==1.1.6
Markdown==3.3.6
MarkupSafe==2.0.1
mock==4.0.3
more-itertools==8.12.0
multidict==5.2.0
natsort==8.0.2
netifaces==0.11.0
oauthlib==3.1.1
olefile==0.46
packaging==21.3
pbr==5.8.0
pexpect==4.8.0
Pillow==8.4.0
ptyprocess==0.7.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycairo==1.20.1
pycparser==2.21
pycrypto==2.6.1
Pygments==2.10.0
PyGObject==3.42.0
PyHamcrest==2.0.3
pyinotify==0.9.6
PyJWT==2.3.0
pymacaroons==0.13.0
PyNaCl==1.4.0
pyOpenSSL==21.0.0
pyparsing==3.0.6
pyrsistent==0.18.0
pyserial==3.5
python-apt==2.0.1+ubuntu0.20.4.1
python-debian==0.1.42
pytz==2021.3
pyxdg==0.27
PyYAML==6.0
requests==2.26.0
requests-unixsocket==0.3.0
roman==3.3
SecretStorage==3.3.1
service-identity==21.1.0
simplejson==3.17.6
six==1.16.0
snowballstemmer==2.2.0
sos==4.4
Sphinx==4.3.2
sphinx-rtd-theme==1.0.0
sphinxcontrib-applehelp==1.0.2
sphinxcontrib-devhelp==1.0.2
sphinxcontrib-htmlhelp==2.0.0
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.3
sphinxcontrib-serializinghtml==1.1.5
ssh-import-id==5.11
stig==0.12.2a0
systemd-python==234
testresources==2.0.1
Twisted==21.7.0
typing_extensions==4.0.1
ubuntu-advantage-tools==8001
ufw==0.36
unattended-upgrades==0.1
urllib3==1.26.7
urwid==2.1.2
urwidtrees==1.0.3
wadllib==1.3.6
yarl==1.7.2
zipp==3.6.0
zope.interface==5.4.0 
```
</details>

**Comment by Hollier** [comment]
I am getting is as well. any movement on this?

**Comment by gazpachoking** [comment]
What version of transmission-rpc are you on? I think this is related to #3715 Try to update transmission-rpc to 3.4.2

**Comment by antivirtel** [comment]
Thanks, that worked. I guess you don't want to make it a requirement for flexget (via pip), so maybe catch this exception in the code to display a relevant error message? 

Can you also please place something about this to the upgrade actions or so? https://flexget.com/UpgradeActions

It's also out of date here: https://flexget.com/Plugins/transmission - I have v3 Transmission, so I suppose I have to use 'transmission-rpc<4' (so it doesn't get out of date in the docs). Thank you!

**Comment by gazpachoking** [comment]
All good ideas. There was some confusion when the changes were made and we didn't realize it had changed our requirements. Anyone should be able to edit those wiki pages, would be great if someone could make those updates!

**Comment by trim21** [comment]
this should be fixed in flexget 3.6.0，please upgrade rpc to v4
